index.html

```diff
@@ -35,5 +35,156 @@ <h2>I'm a Software Developer</h2>
+                <li><a href="#">RUBY</a></li>
+            </ul>
+            <a class="see-project" href="#">See project</a>
+        </div>
+        <div class="card-3">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads;
+                no accounts or sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="link-1">
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">RUBY</a></li>
+            </ul>
+            <a class="see-project" href="#">See project</a>
+        </div>
+        <div class="card-3">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads;
+                no accounts or sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="link-1">
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">RUBY</a></li>
+            </ul>
+            <a class="see-project" href="#">See project</a>
+        </div>
+        <div class="card-3">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads;
+                no accounts or sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="link-1">
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">RUBY</a></li>
+            </ul>
+            <a class="see-project" href="#">See project</a>
+        </div>
+        <div class="card-3">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads;
+                no accounts or sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="link-1">
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">RUBY</a></li>
+            </ul>
+            <a class="see-project" href="#">See project</a>
+        </div>
+        <div class="card-3 last">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads;
+                no accounts or sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="link-1">
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">RUBY</a></li>
+            </ul>
+            <a class="see-project" href="#">See project</a>
+        </div>
+    </section>
+    <section class="about-myself">
+        <h1>About me</h1>
```

Comment:
- [ ] For semantic reasons I will bring to your notice that it is recommended to have just a single `<h1>` tag throughout an HTML file which represents the main title of the website. Other headings should be` h2 - h6` which is the order of the hierarchy of headings.

Therefore it will be better you change your `<h1> About me </h1>` to `<h2> About me </h2>` to increase the accessibility of your website 👍🏽